contracts/seed/Seed.sol

```diff
@@ -316,32 +283,46 @@ contract Seed {
     /**
       * @dev                     Close distribution.
     */
-    function close() public onlyAdmin isActive {
+    function close() external onlyAdmin {
+        // close seed token distribution
+        require(!closed, "Seed: should not be closed");
+        closed = true;
+        paused = false;
+    }
+
+    /**
+      * @dev                     retrieve remaining seed tokens back to project.
+      * @param _refundReceiver   refund receiver address
+    */
+    function retrieveSeedTokens(address _refundReceiver) external onlyAdmin {
         // transfer seed tokens back to admin
-        if (minimumReached) {
-            // remaining seeds = seedRemainder + feeRemainder
-            uint256 seedToTransfer = seedRemainder+feeRemainder;
+        require(
+            closed,
```

Comment:
Doesn't need to be closed.  Should be possible at any point where buying is no longer possible.

Reply:
buy() is inactive in four conditions,
1. when closed == true
2. when maximumReached== true, i.e. no seed tokens remaining
3. when block.timestamp > endTime, i.e. after the distribution end time
4. when paused == true.

We should only allow admin to retrieve seeds tokens only after first three conditions. Not when seed is paused.

So here are the condition when admin can retrieve seed tokens,
1. closed == true
2. block.timestamp > vestingStartTime. here vestingStartTime is the timestamp for when vesting starts, by default == endTime, otherwise when maximumReached is reached

In both condition buy() is unavailable.

Reply:
Why not when the seed is paused?

Reply:
I stand by what I said:  Should be possible at any time after buying is no longer possible.

That means  require(closed || maxReached || (minReached && (blocktime >= endTime)))

What is wrong with this?

Reply:
done